lib/entry/LiveEntry.js

```diff
@@ -302,6 +302,7 @@ LiveEntry.prototype.entryObjectUpdated = function(entry)
         && origViewMode != kalturaTypes.KalturaViewMode.ALLOW_ALL)
     {
         this._startDVR(entry);
+        this._setClipFrom(entry.updatedAt * 1000, false);
```

Comment:
this._setClipFrom(entry.updatedAt-config.get('minDurationForPlaybackInSec') * 1000 , false);